src/components/label/appsco-resource-add-labels.html

```diff
@@ -283,6 +293,34 @@
                 }
             },
 
+            _computeTitle: function (resourceType) {
```

Comment:
1. No need for brake after return. Also, if you have the same return result for different cases you can do this:
========
case 'accounts':
case 'contacts':
    return 'Add to group.'
========

2. There should be 'default' expression state
=======
case 'expression':
    code block
default:
    return false OR return '' OR return 'Share to group'; 

https://www.w3schools.com/js/js_switch.asp